packages/web/src/app/doc-pages/components/component-data.interface.ts

```diff
@@ -31,6 +31,7 @@ interface PropBase {
  */
 export interface ChildlessProp extends PropBase {
   isEvent?: boolean;
+  isFunction?: boolean;
```

Comment:
Since a prop cannot be both an event and a function at the same time, maybe it would be more appropriate to refactor the `isEvent` prop to be something like
`specialType?: 'event' | 'function'`?

Reply:
An alternative approach would be to make a different type `ChildlessFunctionProp` and `ChildlessEventProp` and `ChildlessRegularProp` and then define `ChildlessProp` as
``` ts
type ChildlessProp = ChildlessFunctionProp | ChildlessEventProp | ChildlessRegularProp;
```

Reply:
The first suggestion is far easier though, but you be the judge 🙂

Reply:
After thinking about it; is a function childless? In the case of `toast`, it is not childless, so it should be a `NestedProp`. With the header though, the method is childless (since it takes no arguments), so I guess we need to support both.